ReactiveCocoa/Swift/Signal.swift

```diff
@@ -331,6 +331,21 @@ extension SignalType {
 	}
 }
 
+extension SignalType {
+	/// Merges the given signals into a single `Signal` that will emit all values
+	/// from each of them, and complete when all of them have completed.
+	public static func mergeSignals(signals: [Signal<Value, Error>]) -> Signal<Value, Error> {
```

Comment:
I had no idea this was possible, but I just tried

```swift
func merge<S : SequenceType where S.Generator.Element == Signal<Value, Error>>(signals: S) -> Signal<Value, Error>
```

… and that worked? Swift. 🙃